### ISSUE TITLE
推理代码有BUG

### ISSUE THREAD

**Issue by JasonCZH4** [zhaogaofeng611/TextMatch#15]
在`test.py`文件中，`DataLoader(test_data, shuffle=True, batch_size=batch_size)` shuffle应该设为False，这是一个很低级的BUG，卡了我一天，在最意想不到的地方有BUG
